hudi-common/src/main/java/org/apache/hudi/common/config/HoodieCommonConfig.java

```diff
@@ -78,18 +78,18 @@ public class HoodieCommonConfig extends HoodieConfig {
 
   public static final ConfigProperty<String> INCREMENTAL_READ_HANDLE_HOLLOW_COMMIT = ConfigProperty
       .key("hoodie.datasource.read.handle.hollow.commit")
-      .defaultValue(HollowCommitHandling.EXCEPTION.name())
+      .defaultValue(HollowCommitHandling.FAIL.name())
       .sinceVersion("0.14.0")
       .markAdvanced()
       .withValidValues(enumNames(HollowCommitHandling.class))
       .withDocumentation("When doing incremental queries, there could be hollow commits (requested or inflight commits that are not the latest)"
```

Comment:
lets ensure we call out time travel queries as well. 
```
When doing incremental queries or time travel queries, there ... 
```

Reply:
time travel query related change is in a follow up PR. this one is meant only for renaming